go/test/endtoend/backup/vtctlbackup/backup_utils.go

```diff
@@ -800,15 +804,9 @@ func terminatedRestore(t *testing.T) {
 // Args:
 // tablet_type: 'replica' or 'rdonly'.
 func vtctlBackup(t *testing.T, tabletType string) {
-	// Start vtorc before running backups
```

Comment:
This was creating multiple instances of VTorc during testing. I don't think this was intentional. If we want to test multi instance VTorc then we should write a separate test for it. This is backup test and should try to test backup functionality.

Reply:
Yes, this wasn't intentional.